### PR TITLE
added multi-variable and large responses links back in

### PR DESCRIPTION
**Pull request by PatrickAlphaC**

### CODE REVIEW

**Comment by StephenFluin** [comment]
@dwightjl need to figure out how it got removed and see where the process is failing if it is

File: _data/navigation.js [hunk 1]
```diff
@@ -239,6 +239,16 @@ module.exports = {
             title: 'Make an Existing Job Request',
             url: '/docs/existing-job-request/',
           },
+          ,
```
Comment (StephenFluin) [inline _data/navigation.js:242]: I think this comma is extra

File: _data/navigation.js [hunk 2]
```diff
@@ -239,6 +239,16 @@ module.exports = {
...
+          {
+            title: 'Large Responses',
+            url: '/docs/large-responses/',
+          },
+          ,
```
Comment (StephenFluin) [inline _data/navigation.js:247]: This one should be removed too

**Review by StephenFluin — CHANGES_REQUESTED** [review]
Remove the extra commas and we're good to go

**Comment by dwightjl** [comment]
> @dwightjl need to figure out how it got removed and see where the process is failing if it is

Looks like the changes to navigation.js got clobbered in 23ec285 right before PR #270 merged.    https://github.com/smartcontractkit/documentation/commit/23ec285b36b23db8ed8822e643f229df264fbb99

I've made that mistake before.

**Comment by dmitrydao** [comment]
replaced with #302, which is already approved

**Comment by dwightjl** [comment]
@dmitrydao Closing this PR is ok, but in the future let's keep duplicate PRs open until one of them is merged. Approval from me is not the same as merging or closing the PR.

**Pull request closed without merging**